.github/workflows/maven.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous Integration Setup
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'corretto'
+          java-version: '21'
+          cache: 'maven'
+
+      - name: Clean install
+        run: mvn clean install
+      
+      - name: Execute Unit Tests
```

Comment:
Do we need this extra step? `mvn clean install` in the previous step also runs the unit tests.

Reply:
Removed in commit [492daeb](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/492daeb48360470829cc54218911b9b3e1758ace)